pi5/board.go

```diff
@@ -66,7 +66,7 @@ func newBoard(
 
 	var physical_address uint64 = 0
 	var virtual_address uint64 = 0
-	var gpiomem_idx [4]rune
+	var gpioMemIdx [4]rune
```

Comment:
I had to look up what a rune was, is there a reason you cant use `int32`? if it doesnt work with int32 can you put a comment explaining why you need to use rune.

Reply:
I initially set this to rune because I thought I could use it to represent a collection of bits, but runes are actually used to represent individual chars in go. However, this variable is unused in address translation so I'll remove it from the PR.

Reply:
reading up more about rune, and I could also just use string for this and have the strings have a length of 1.